src/app.ts

```diff
@@ -1,8 +1,11 @@
 import dotenv from 'dotenv';
+dotenv.config();
+
 import express from 'express';
-import mainRouter from './api/mainRouter';
+import 'express-async-errors';
+import mainRouter from './api';
 
-dotenv.config();
+import './loaders/sequelize';
```

Comment:
it would be better to create separated module `./loaders/index.ts` and place all dependencies there.

Reply:
done